src/components/DocumentationTopic/PrimaryContent/DeclarationDiff.vue

```diff
@@ -17,7 +17,6 @@
         :key="i"
         :declaration="declaration"
         :should-caption="currentDeclarations.length > 1"
-        :changeType="changeType"
```

Comment:
Should we remove `changeType` as a prop for this component? It doesn't seem like it's being used anymore.